src/app/page.tsx

```diff
@@ -12,15 +11,15 @@ export default function Home() {
           width={1500}
           height={850}
           src={bannerImage}
-          alt="banner-image.png"
+          alt="bars making a smooth 90 degree curve in olympic colors"
```

Comment:
Thanks for adding these alt texts. Two requests: 
 
 - Please capitalize the first letter of the first word of each sentence (this text will be visible if the image fails to load).
 - Avoid making references to gender. Simply saying "a person" is enough to get the point across without making any assumptions.